src/utils/jobData.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: Add real job data
+const jobData = [
+    {
+        title: 'Software Engineer',
+        company: 'Amazon Web Services (AWS)',
+        logoUrl: '/',
+        location: 'Tokyo, Japan',
+        jobPostingUrl: '#',
```

Comment:
Example.com is always a good default for a generic website. https://placebear.com/ is good for test images

```suggestion
        jobPostingUrl: 'http://example.com',
```